kythe/javatests/com/google/devtools/kythe/analyzers/java/testdata/DefaultPackage.java

```diff
@@ -1,14 +1,14 @@
 // Tests for a class in the default package. All name nodes should be prefixed with a ".", denoting
 // the default package.
 
-//- @DefaultPackage defines/binding Class
-//- !{ Class childof AnyPackage
-//-    AnyPackage.node/kind package }
+// - @DefaultPackage defines/binding Class
+// - !{ Class childof AnyPackage
+// -    AnyPackage.node/kind package }
```

Comment:
I can't recall if the verifier goal regex allows for this?

Reply:
https://github.com/kythe/kythe/blob/bca72abb2455ddfe2f8d355e17c545bd084d0a97/tools/build_rules/verifier_test/java_verifier_test.bzl#L232